libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/couchdb/AttachmentStreamConnector.java

```diff
@@ -133,6 +146,20 @@ public InputStream unsafeGetAttachmentStream(AttachmentContent attachment) throw
         return in;
     }
 
+    private String printAcceptedFileName(String name, int count) {
+        String acceptedFileName;
+        int extensionDot;
```

Comment:
why not join declaration and assingment?

Reply:
changed to direct assignment. ( `getExtensionFromFileName` )